modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1302,36 +1312,55 @@ RETURN val, val IS NOT :: STRING AS notString
 [[type-predicate-expressions-null]]
 === Type predicate expressions for null
 
-All Cypher types include the `null` value.
-Therefore, `IS ::` will return `true` for all expressions evaluating to `null`, regardless of the type.
+A Cypher type will include the `null` value unless it is prepended explicitly with `NOT NULL`.
```

Comment:
not `prepended` I think, I believe that would be something like `NOT NULL INTEGER` instead of `INTEGER NOT NULL`. I think maybe `appended` (`suffixed`?) or something is a better choice